src/main.cpp

```diff
@@ -101,13 +102,17 @@ int main() {
     glm::vec3(0.0f, 0.0f, 10.0f),
     Color(0x888888),
     {
-      {Color(0xFFFFFF), glm::vec3(0.5, 0.0, -1.0)}
+      {Color(0xFFDD33), glm::vec3(0.5, 0.0, -1.0)},
+      {Color(0x330066), glm::vec3(-3, 0.0, 2.0)}
     }
   );
 
   while(!glfwWindowShouldClose(window)) {
     processInput(window);
 
+    rect.setRotation(glm::vec3(0, glfwGetTime(), glfwGetTime()));
+    rect2.setRotation(glm::vec3(glfwGetTime(), glfwGetTime(), 0));
```

Comment:
Technically this may skew a TINY bit. But it may be better to store the current time in a variable to ensure that all these times are the same. :)